### PR TITLE
Add GPROFNF_SYSCONFDIR to gcc-cxxcompiler toolfile on AL9 and above

### PR DESCRIPTION
**Pull request by gartung**

### CODE REVIEW

**Comment by gartung** [comment]
please test

File: scram-tools.file/tools/gcc/gcc-cxxcompiler.xml [hunk 1]
```diff
@@ -5,6 +5,9 @@
       <environment name="CXX" value="$GCC_CXXCOMPILER_BASE/bin/c++@COMPILER_NAME_SUFFIX@"/>
 %else
       <environment name="CXX" value="c++@COMPILER_NAME_SUFFIX@"/>
+%endif
+if 0%{?rhel} >= 9
+<runtime name="GPROFNG_SYSCONFDIR" value="$GCC_CXXCOMPILER_BASE/etc"/>
```
Comment (smuzaffar) [inline scram-tools.file/tools/gcc/gcc-cxxcompiler.xml]: @gartung , please move this outside the `<client></client>` section. I would suggest to add this around line 43 (also properly align it with other `<runtime name` statements)

Comment (gartung) [reply scram-tools.file/tools/gcc/gcc-cxxcompiler.xml]: OK

**Comment by smuzaffar** [comment]
please test

**Comment by gartung** [comment]
Should I move it outside of the `%if 0%{!?use_system_gcc:1}` block?

**Comment by smuzaffar** [comment]
> Should I move it outside of the `%if 0%{!?use_system_gcc:1}` block?

No, it is better to keep it inside this block.

File: scram-tools.file/tools/gcc/gcc-cxxcompiler.xml [hunk 2]
```diff
@@ -41,6 +41,9 @@
     <runtime name="@OS_RUNTIME_LDPATH_NAME@" value="$GCC_CXXCOMPILER_BASE/lib" type="path"/>
     <runtime name="COMPILER_PATH" value="@TOOL_ROOT@"/>
     <runtime name="PATH" value="$GCC_CXXCOMPILER_BASE/bin" type="path"/>
+if 0%{?rhel} >= 9
```
Comment (smuzaffar) [inline scram-tools.file/tools/gcc/gcc-cxxcompiler.xml]: @gartung , you are missing `%` here . it should be `%if `

Comment (gartung) [reply scram-tools.file/tools/gcc/gcc-cxxcompiler.xml]: Yes. Sorry.

**Comment by gartung** [comment]
please test

**Pull request merged**